lib/index.js

```diff
@@ -39,6 +39,9 @@ class WatsonMiddleware {
         if (config.minimum_confidence) {
             this.minimumConfidence = config.minimum_confidence;
         }
+        if(config.storage){
+            this.storage = config.storage;
+        }
```

Comment:
Javascript files in `lib/` directory are generated from Typescript files in `src/`.
Your changes will be lost on the next rebuild.

Please update src/index.ts instead and run `npm run build`.

Reply:
@Naktibalda Done !